recipes/dmox/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "0.1.1" %}
+{% set sha256 = "d08efd4fc59be9a481d24e3f6ae611a343e6d03fa49552c898901ba3b55896de" %}
```

Comment:
Is this supposed to be a commit in `dmox`'s history @pdimens ? I think there is no such commit there. The release is on `783543825a465150c9fa0dd08e2e550a3ba9625e`.

Reply:
It's the sha256 for the `.tar.gz` file of the `0.1.1` release. You only really need to manually set this once-- once `dmox` is merged into Bioconda, the bot will automatically change the version number and set the appropriate sha256 and submit a PR on your behalf to Bioconda with all the relevant details.

Reply:
Oh I see it's clearer now. Thank you :)

Reply:
So, to clarify, you are right, there is no commit `d08efd4fc59be9a481d24e3f6ae611a343e6d03fa49552c898901ba3b55896de`, the checksum is specifically for **that gzipped tarball of the release**